src/app/pages/reports-dashboard/components/report/report.component.ts

```diff
@@ -384,12 +384,17 @@ export class ReportComponent extends WidgetComponent implements OnInit, OnChange
   }
 
   handleError(evt: CoreEvent): void {
+    const err = evt.data.data;
+    if (evt.data?.data?.error === ReportingDatabaseError.FailedExport) {
+      this.report.errorConf = {
+        type: EmptyType.Errors,
+        title: this.translate.instant('Error getting chart data'),
+        message: err.reason,
+      };
+    }
```

Comment:
Should we return here? Or create an object variable which holds the `errorConf` and assign it only once at the end of the method. Whichever seems a better option.

Reply:
Thanks for the suggestions, and sorry if the PR title is confusing. It is a bug fix to prevent breaking the UI in specific conditions—no plans for refactoring in this PR.